subprojects/docs/src/snippets/java/toolchain-management/kotlin/settings.gradle.kts

```diff
@@ -0,0 +1,77 @@
+import org.gradle.api.Incubating
+import org.gradle.api.Plugin
+import org.gradle.api.initialization.Settings
+import org.gradle.jvm.toolchain.JavaToolchainResolver
+import org.gradle.jvm.toolchain.JavaToolchainResolverRegistry
+import java.net.URI
+import java.util.Optional
+import javax.inject.Inject
+
+apply<AzulPlugin>()
+apply<AdoptiumPlugin>()
+
+// tag::toolchain-management[]
+toolchainManagement {
+    jvm { // <1>
+        javaRepositories {
+            repository("azul") { // <2>
+                resolverClass.set(AzulResolver::class.java)
+                credentials {
+                    username = "user"
+                    password = "password"
+                }
+                authentication {
+                    create<DigestAuthentication>("digest")
+                } // <3>
+            }
+            repository("adoptium") { // <4>
+                resolverClass.set(AdoptiumResolver::class.java)
+            }
+        }
+    }
+}
+// end::toolchain-management[]
+
+rootProject.name = "toolchain-management"
+
+/**
+ * Mock Azul repository plugin.
+ *
+ * @since 7.6
+ */
+@Incubating
+abstract class AzulPlugin: DummyPlugin(AzulResolver::class)
+
+/**
+ * Mock Adoptium repository plugin.
+ *
+ * @since 7.6
+ */
+@Incubating
```

Comment:
:x: Javadoc and `@Incubating` are not needed in snippets, they should be removed